README.md

```diff
@@ -68,7 +68,8 @@ The available options are:
   --source-map-inline           Write base64-encoded source map to the end of js output.
   --in-source-map               Input source map, useful if you're compressing
                                 JS that was generated from some other original
-                                code.
+                                code. Specify "inline" if the source map is included
+				inline with the sources.
```

Comment:
Don't know why the diff is showing incorrect indentation level here, it's fine in the source.